src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.Core.csproj

```diff
@@ -1,19 +1,21 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Repo.props" />
   <PropertyGroup>
     <LangVersion>9.0</LangVersion>
     <TargetFrameworks>netstandard2.0;net6.0;net472</TargetFrameworks>
     <RootNamespace>Microsoft.Graph.PowerShell.Authentication.Core</RootNamespace>
-    <Version>2.18.0</Version>
+    <Version>2.12.0</Version>
   </PropertyGroup>
   <PropertyGroup>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.11.4" />
-    <PackageReference Include="Azure.Identity.Broker" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Graph.Core" Version="3.0.9" />
+    <PackageReference Include="Azure.Identity" Version="1.12.0-beta.3" />
+    <PackageReference Include="Azure.Identity.Broker" Version="1.2.0-beta.1" />
```

Comment:
In previous conversations, we had wondered if Azure.Identity is actually bringing any value here over using Microsoft.Identity.Client directly.  Azure.Identity is intended to limit the surface area exposed to 3P developers. This is not relevant for PowerShell scenarios.  I am especially concerned when I see the introduction of Azure.Core.Pipeline. We explicitly don't use the Azure Pipelining because it is only there to abstract away language differences, and again that isn't relevant because our PowerShell modules are only implemented in C#.